cli/templates.mjs

```diff
@@ -163,9 +163,53 @@ describe('${upperName}', () => {
   }
 }
 
+const doc = ({ name }) => {
+  const kebabName = kebabCase(name)
+  const upperName = splitByCase(name).map(p => upperFirst(p)).join('')
+
+  return {
+    // TODO: change the path after adding PRO components
+    filename: `docs/content/3.components/${kebabName}.md`,
```

Comment:
It's a bit complex for UI Pro as folders will not be based on path but grouped by "categories".

I guess you can put them in `docs/content/pro/` and I'll move it in the right category manually 😊